src/ResourceManager/ContainerInstance/Commands.ContainerInstance/Commands/NewAzureContainerGroupCommand.cs

```diff
@@ -0,0 +1,163 @@
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        [Parameter(
+           Mandatory = true,
+           HelpMessage = "The container image.")]
+        [ValidateNotNullOrEmpty]
+        public string Image { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The container group Location. Default to the location of the resource group.")]
+        [ValidateNotNullOrEmpty]
+        public string Location { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The container OS type. Default: Linux")]
+        [ValidateNotNullOrEmpty]
+        [ValidateSet(
+            OperatingSystemTypes.Linux,
+            OperatingSystemTypes.Windows,
+            IgnoreCase = true)]
+        public string OsType { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The required CPU cores. Default: 1")]
+        [ValidateNotNullOrEmpty]
+        public double? Cpu { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The required memory in GB. Default: 1.5")]
+        [ValidateNotNullOrEmpty]
+        public double? Memory { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The IP address type.")]
+        [ValidateNotNullOrEmpty]
+        [ValidateSet(
+            "Public",
+            IgnoreCase = true)]
+        public string IpAddressType { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The port to open.")]
+        [ValidateNotNullOrEmpty]
+        public int? Port { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The command to run in the container. e.g. @(\"executable\",\"param1\",\"param2\")")]
+        [ValidateNotNullOrEmpty]
+        public string[] Command { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The container environment variables.")]
+        [ValidateNotNullOrEmpty]
+        public Hashtable EnvironmentVariables { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The custom container registry login server.")]
+        [ValidateNotNullOrEmpty]
+        public string RegistryServer { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The custom container registry username.")]
+        [ValidateNotNullOrEmpty]
+        public string RegistryUsername { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The custom container registry password.")]
+        [ValidateNotNullOrEmpty]
+        public SecureString RegistryPassword { get; set; }
+
+        [Parameter(
+           Mandatory = false,
+           ValueFromPipelineByPropertyName = true)]
+        [Alias("Tag")]
+        public Hashtable Tags { get; set; }
```

Comment:
@yolo3301 this should be called `Tag` instead of `Tags`

Reply:
Changed Tag